bin/mocha

```diff
@@ -45,13 +50,12 @@ const disableTimeouts = value => {
  * @ignore
```

Comment:
above L40:
`mochaArgs.timeout` is a number, should be set to 0?

Reply:
when it's `false` it gets set to `0`, but might as well make it `0`.